scripts/gbp_format_changelog

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Simple changelog entry formatter
+#
+# It simply uses the built in formatter and linewraps the text
+#
+# Use git-dch --customizations=/usr/share/doc/git-buildpackage/examples/wrap_cl.py
+# or set it via gbp.conf
+
+import textwrap
+import re
+import gbp.dch
+
+
+FILTER_CLOUD_INIT_UPSTREAM_COMMITERS = (
+    "Chad Smith", "James Falcon", "Brett Holman"
+)
+
+FILTER_NOISY_COMMIT_REGEX = (
+  r'update changelog.*',
+  r'refresh patches against.*',
+)
+
+UNWRAPPABLE_DELIMITERS = {
+  "]": "[",
+  ")": "(",
+}
+
+
+def _wrap_on_delimiter(text, prefix="", max_length=70) -> list:
+    """Break lines at specific UNWRAPPABLE_DELIMITERS.
+
+    When a line ends with either (LP: #XXX) or [Author Name] avoid using
+    textwrap.wrap which breaks at the last whitespace.
+
+    Instead break at the leading ( or [ special delimiter to ensure entire
+    author name or LP bug reference remains on the same line.
+
+    Fallback to use textwrap.wrap if special conditions don't apply.
+
+    Return a list of individual lines.
+    """
+    if len(text) <= max_length:
+        return [prefix + text]
+    if text[-1] in UNWRAPPABLE_DELIMITERS:
+        delimiter = UNWRAPPABLE_DELIMITERS[text[-1]]
+        part1, sep, part2 = text.rpartition(delimiter)
+        remainder = ""
+        lines = []
+        for part in (part1.rstrip(), f"{sep}{part2}"):
+            if lines:
+                if len(lines[-1] + " " + part) < 70:
+                    # Then the previous part plus current part should be joined
+                    part = lines.pop() + " " + part
+                part = f" {part}"
+            lines.extend(
+                _wrap_on_delimiter(part, prefix = "" if lines else "+")
+            )
+        return lines
+    return textwrap.wrap(prefix + text)
+
+
+def format_changelog_entry(commit_info, options, last_commit=False):
+    entry = gbp.dch.format_changelog_entry(commit_info, options, last_commit)
+    if re.search(r"\(SC-\d+\)", entry[0]):
```

Comment:
Worth putting `SC` in a global variable? `JIRA_PROJECT_KEY = "SC"` (I think SC- is called a "project key").